src/core/StelUtils.cpp

```diff
@@ -103,6 +103,30 @@ double hmsStrToHours(const QString& s)
 	return hmsToHours(hrs, min, sec);
 }
 
+/*************************************************************************
+ Convert a real duration in days to DHMS formatted string
+*************************************************************************/
+QString daysFloatToDHMS(float days)
+{
+	float remain = days;
+
+	int d = remain; remain -= d;
```

Comment:
Please fix type cast issue

Reply:
you mean this?
```C++
int d = (int) remain;
int d = static_cast<int> (remain);
```

Reply:
Yes, I mean `int d = static_cast<int> (remain);`

Reply:
ok